modules/mailer/mailer.go

```diff
@@ -209,6 +209,7 @@ func (s *sendmailSender) Send(from string, to []string, msg io.WriterTo) error {
 	var waitError error
 
 	args := []string{"-F", from, "-i"}
+	args = append(args, setting.MailService.SendmailArgs)
```

Comment:
See my comment above

Reply:
Just splitting it seems to be good enough unless nothing to fancy pancy? Same is done for args to GC and health check.